core/modules/system/system.module

```diff
@@ -4561,3 +4561,59 @@ function system_theme_debug_enabled_warning() {
     backdrop_set_message(t('Theme debugging is enabled. See the <a href="!url">Status report</a> page for more info.', array('!url' => url('admin/reports/status'))), 'warning', FALSE);
   }
 }
+
+/**
+ * Wrapper around transliteration_get() that handles common clean up on strings.
+ *
+ * @param string $string
+ *   The string to be transliterated.
+ * @param $options
+ *   An array of options for further cleanup. This array includes:
+ *   - langcode: The language code of the source string.
+ *   - replace: The character with which to replace unknown characters in the
+ *     string. Defaults to '_'.
+ *   - replace_pattern: A pattern for replacing unknown characters with the
+ *     "replace" value.
+ *   - maxlength: The maximum allowed length of the returned string.
+ * @return string
```

Comment:
Can we please add a blank line to separate parameters from the return value?:
```suggestion
 *
 * @return string
```

Reply:
Those were preexisting. If @najmieh has time to do so, otherwise this can be a followup issue to work on core docblocks generally.

Reply:
Yes, I can do it.

> Can we please add a blank line to separate parameters from the return value?:
I need to work on this?